### PR TITLE
Remove unused database triggers and functions

### PR DESCRIPTION
**Pull request by dnephin**
## Summary

Branched from #4042, this PR removes the now unused triggers and functions.

We should deploy this as a separate change to not break active long-polling requests.

### CODE REVIEW

File: internal/server/data/migrations.go [hunk 1]
```diff
@@ -1173,3 +1174,33 @@ func addUserPublicKeyUserIDIndex() *migrator.Migration {
 		},
 	}
 }
+
+func sqlFunctionsMigration() *migrator.Migration {
```
Comment (dnephin) [inline internal/server/data/migrations.go:1178]: This moved from another file, so that I could delete the now empty file.

**Review by BruceMacD — COMMENTED** [review]
This seems good once the other changes go in. If this deployed with connections actively waiting on these triggers they would just return, then a new request would open another poll that would work with the new polling triggers, correct?

**Comment by dnephin** [comment]
I don't think active requests will notice the removal of triggers, so I'd expect the active requests to block until the timeout, then repeat the request and be fine.

**Review by BruceMacD — APPROVED** [review]

**Comment by mxyng** [comment]
Closing this for now

**Pull request closed without merging**